framework/src/media/FileOutputDataSource.cpp

```diff
@@ -16,6 +16,8 @@
  *
  ******************************************************************/
 
+#include <debug.h>
+
```

Comment:
Why this is required  when no debug is enabled? I assume this header should be included only for debugging and not used in normal situation.

Reply:
Adding the header to fix build error is not right approach. Please identify the correct reason for error and handle it accordingly. May be you are missing a config option

Reply:
Without this code, build failure on Linux. You can check all related header files. There is no debug.h but in the code, we are using 'medvdbg'. TizenRT build is not strict.
MediaPlayer should be built on TizenRT and Linux both.